### DISCUSSION TITLE
Projects overview - "open exceptions" indication

### DISCUSSION THREAD

**Discussion by fabricecw** [spatie/flareapp.io-roadmap#45] [Ideas]
The projects overview shows the amount of unresolved exceptions per project. If you open a project, only unresolved and unsnoozed exceptions are listed by default.

We have a quite a few exceptions which are snoozed and just "for monitoring" (external API retries, OAuth authentication exceptions, ...).

Currently, we need to jump into projects to ensure that there is no untreated / open exception since snoozed exceptions are counted in the projects overview.

In our eyes, the overview's approach should be "is there something to do?" - so an indication of unsnoozed & unresolved errors (same as the default filter in a project). Maybe indicated as "Open".

**Comment by freekmurze** [comment]
Thanks for your suggestion. So summarised: you'd like snoozed exceptions not to be counted as an unresolved exception, right?

**Comment by fabricecw** [comment]
Yep, maybe with another name than "unresolved" in the overview to avoid confusion.

**Comment by dmason30** [comment]
Also would like to see this changed. Best way to avoid confusion would probably to squeeze in a separate snoozed column 👍

**Comment by AlexVanderbist** [comment]
The project overview now shows the truly unresolved errors (also considering snoozed errors)!
